CHANGELOG.md

```diff
@@ -0,0 +1,223 @@
+# Changelog
+
+## Unreleased
+
+## v0.4.0
+
+Released 17-06-2020
+
+# 🎉 OpenTelemetry Collector Contrib v0.4.0 (Beta) 🎉
+
+The OpenTelemetry Collector Contrib contains everything in the [opentelemetry-collector release](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.4.0) (be sure to check the release notes here as well!). Check out the [Getting Started Guide](https://opentelemetry.io/docs/collector/about/) for deployment and configuration information.
+
+## 🛑 Breaking changes 🛑
+
+  - `signalfx` exporter `url` parameter changed to `ingest_url` (no impact if only using `realm` setting)
+
+## 🚀 New components 🚀
+
+- Receivers
+  - `receiver_creator` to create receivers at runtime (#145), add observer support to receiver_creator (#173), add rules support (#207), add dynamic configuration values (#235) 
+  - `kubeletstats` receiver (#237) 
+  - `prometheus_simple` receiver (#184) 
+  - `kubernetes-cluster` receiver (#175) 
+  - `redis` receiver (#138)
+- Exporters
+  - `alibabacloudlogservice` exporter (#259) 
+  - `SplunkHEC` metrics exporter (#246)
+  - `elastic` APM exporter (#240)
+  - `newrelic` exporter (#229) 
+- Extensions
+  - `k8s` observer (#185) 
+
+## 💡 Enhancements 💡
+
+- `awsxray` exporter
+  - Use X-Ray convention of segment name == service name (#282)
+  - Tweak xray export to improve rendering of traces and improve parity (#241)
+  - Add handling for spans received with nil attributes (#212)
+- `honeycomb` exporter
+  - Use SendPresampled (#291)
+  - Add span attributes as honeycomb event fields (#271)
+  - Support resource labels in Honeycomb exporter (#20)
+- `k8s` processor
+  - Add support of Pod UID extraction to k8sprocessor (#219)
+  - Use `k8s.pod.ip` to record resource IP instead of just `ip` (#183)
+  - Support same authentication mechanism as other kubernetes components do (#307)
+- `sapm` exporter: Add TLS for SAPM and SignalFx receiver (#215)
+- `signalfx` exporter
+  - Add metric metadata syncer to SignalFx exporter (#231)
+  - Add TLS for SAPM and SignalFx receiver (#215)
+- `stackdriver` exporter: Add support for resource mapping in config (#163)
+
+## 🧰 Bug fixes 🧰
+
+- `awsxray` exporter: Wrap bad request errors for proper handling by retry queue (#205)
+- `lightstep` exporter: Ensure Lightstep exporter doesnt crash on nil node (#250)
+- `sapm` exporter: Do not break Jaeger traces before sending downstream (#193)
+- `k8s` processor: Ensure Jaeger spans work in passthrough mode (262)
+
+## 🧩 Components 🧩
```

Comment:
Do we need to list all component in change log?

Reply:
We can discuss and remove post-release or in future releases. For now I suggest we continue to use the existing format to not slow things down further :)

Reply:
FWIW, personally I don't see much value in listing down all component every single release either. We can just list down all the new components we have each release. Not changing it right now though.

Reply:
sounds good